src/utilities/index.ts

```diff
@@ -106,6 +106,7 @@ export {
   DangerousChangeType,
   findBreakingChanges,
```

Comment:
I think it might work better to add deprecation notices to the various index files as well — I think in some cases it doesn’t carry through automatically.

Reply:
That's not how tsdoc works though 😅 it's tied to the func def

Reply:
ah it was a bug that they fixed https://github.com/microsoft/TypeScript/issues/53754 i read it recently and must have misunderstood, in theory we can remove some of my extra deprecated comments that I have scattered around, but not a priority